codec-agnostic-rtp-payload-format.md

```diff
@@ -62,6 +62,39 @@ The spatial layers are sent in ascending order, with the same RTP timestamp, and
 If the application wants to encrypt content, the application will split media content into frames before encryption.
 Each frame will be encrypted independently, using {{SFrame}} for instance, and transmitted by the packetizer.
 
+Payload multiplexing
+=======================
+
+In order to reduce the number of payload type codes on the SDP exchange, a single payload type code for the generic packetization will be used for each media type. That requires to also carry the original payload type code for the negotiated media format that the frame belongs to.
```

Comment:
s/will be used/can be used/

Reply:
s/That requires to also carry/That requires to identify the original payload type code for the negotiated media format that the frame belongs to directly from the RTP packets of the frame.

Reply:
changed